internal/services/controller/controller_test.go

```diff
@@ -63,10 +67,22 @@ func TestMain(m *testing.M) {
 func TestController_HappyPath(t *testing.T) {
 	tests := map[string]struct {
 		objectCount int
+		err         error
 	}{
 		"happy path": {
 			objectCount: 14,
 		},
+		"err when fetching api resources - multiple errors": {
+			err: fmt.Errorf("unable to retrieve the complete list of server APIs: %v:"+
+				"stale GroupVersion discovery: some error,%v: another error",
+				policyv1.SchemeGroupVersion.String(), storagev1.SchemeGroupVersion.String()),
+			objectCount: 12,
+		},
```

Comment:
question: can we simplify or make it a little bit easier to understand test cases?

it is too hard to understand what should I expect from the test case name and properties `err`, `objectCount`.